plugins/destination/postgresql/client/types.go

```diff
@@ -68,6 +70,8 @@ func (*Client) SchemaTypeToCockroach(t schema.ValueType) string {
 		return "text[]"
 	case schema.TypeTimestamp:
 		return "timestamp without time zone"
+	case schema.TypeTimeInterval:
+		return "interval" // cockroach defaults to interval(6), so we leave it as is
```

Comment:
why not be explicit and use `interval(6)`?

Reply:
Cocroachdb returns it back as `interval`, so our tests are failing here (pg works fine).

Reply:
Ah okay interesting. Let's leave it as is then, seems unlikely to cause any real trouble.